src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -6203,6 +6229,27 @@ components:
             format: text
             minLength: 1
             maxLength: 1023
+    NodeType:
+      type: string
+      format: enum
+      description:
+        <p>The enum can have three different values</p>
+        <ul>
+          <li>PRIMARY, which is the primary node in a high availability setup</li>
+          <li>REPLICA, which is a replica node in a high availability setup – a read-only server</li>
+          <li>STANDALONE,when there are no load balancer or high availability configured</li>
+        </ul>
+      example: PRIMARY
+      enum:
+        - PRIMARY
+        - REPLICA
+        - STANDALONE
+    NodeTypeResponse:
+      type: object
+      description: Response object containing the node type of the Security Server
+      properties:
```

Comment:
Is the node_type property really optional, or should we have the required - field list here?

Reply:
Yeah, it is required in the response. Fixed